ftl/common/ftl_util.py

```diff
@@ -16,8 +16,19 @@
 import time
 import logging
 import subprocess
-import hashlib
 import tempfile
+import datetime
+import json
+
+_OVERRIDES = ["created", "entrypoint", "env"]
+
```

Comment:
Comments on the whitelist and function here woudl be useful.

Reply:
added